.github/workflows/release.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
       - name: Set up JDK
         uses: actions/setup-java@v2
         with:
-          java-version: '17'
+          java-version: '11'
```

Comment:
Why downgrade? The Java code that lives in this repository was written targeting Java 17 (which is the latest LTS version) so the build job should too, imo. The result will be a .jar file either way, I don't think it matters how many users are on a specific java version as long as they have Java runtime >=9 installed. Correct me if I'm wrong.

Reply:
check https://www.baeldung.com/java-lang-unsupportedclassversion

It should build. But people who use JDK 11 LTS will not be able to load it due to compiled against JDK 17, which has higher version in class file

Reply:
I see. Well, as long as it still builds it's fine I guess.